### PR TITLE
Update OCP-4-13-release-notes for Developer Perspective: Known Issue

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Known Issue Release Notes for Developer Perspective

Openshift 4.13 Release Notes

JIRA issue: [RHDEVDOCS-4868](https://issues.redhat.com/browse/RHDEVDOCS-4868)
Build Preview: https://58236--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-known-issues 

QE Review: @sanketpathak 
Peer Review: @Srivaralakshmi

### CODE REVIEW

**Comment by sanketpathak** [comment]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -1559,6 +1559,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* When adding a Git Repository, the created repository resource was invalid if you configured it with a GitLab and Bitbucket `pipeline-as-a-code` repository. Now, the Git provider URL is removed for GitLab and Bitbucket providers. A mandatory *Bitbucket username* field is added if the provider is Bitbucket. You also need to select either *Git access token* or *Git access token secret* to continue adding a Git Repository. (link:https://issues.redhat.com/browse/OCPBUGS-7036[*OCPBUGS-7036*])
```
Comment (Srivaralakshmi) [inline release_notes/ocp-4-13-release-notes.adoc:1562]: @shreyasiddhartha I see that the issue here is that the created repository resource was invalid. Are the remaining sentences the workaround for this issue? Or they are also part of the Known issue?

Also, I do not see any mention of this: spec.git_provider.user details for BitBucket provider. Any particular reason why this is not mentioned?

Comment (Srivaralakshmi) [reply release_notes/ocp-4-13-release-notes.adoc:1562]: @shreyasiddhartha Also please note that this is the template RH SSG suggests for writing the RN Texts: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts

Please adhere to these guidelines and standards when documenting RN Texts for KIs or KIs with workarounds. Thanks!

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@shreyasiddhartha PTAL at my questions and suggestions.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc]: @Srivaralakshmi I have made a few changes as per your suggestions. PTAL. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -1559,6 +1559,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* Currently, when adding a Git Repository, the created repository resource is invalid if you configure it with a GitLab and Bitbucket `pipeline-as-a-code` repository.
```
Comment (Srivaralakshmi) [inline release_notes/ocp-4-13-release-notes.adoc:1562]: @shreyasiddhartha Requires rewrite. Are the steps sequential here?

Typically, workarounds are written as instructions and in the present tense. Please see the example in the template.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@shreyasiddhartha PTAL at my suggestions. Thanks!

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply release_notes/ocp-4-13-release-notes.adoc:1562]: @shreyasiddhartha Also in the OCPBUGS-7036 description I see this sentence: The spec.git_provider.url should be https://api.bitbucket.org/2.0, or **can be removed completely. Both work fine.**

But the RN text Workaround in your PR and the solution description in the engineering PR #12593 says it is mandatory. I do not find this information in the OCP JIRA. This is confusing. Please consult with the SME and clarify.

Comment (Srivaralakshmi) [reply release_notes/ocp-4-13-release-notes.adoc:1562]: @shreyasiddhartha Is the `spec.git_provider.user` field and the **Bitbucket username** field different from each other? Both are mandatory? Please consult the SMEs and confirm.

It seems that they are the same for me, but not sure.

Comment (shreyasiddhartha) [reply release_notes/ocp-4-13-release-notes.adoc]: @Srivaralakshmi I have made some changes after consulting with the SME. PTAL and let me know if this looks good.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -1622,6 +1622,9 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* Currently, when adding a Git Repository, the created repository resource is invalid if you configure it with a GitLab and Bitbucket `pipeline-as-a-code` repository. As a result, the Git provider URL `spec.git_provider.url` is removed for GitLab and Bitbucket providers. With this workaround, you use the mandatory *Bitbucket username* field for Bitbucket and select either *Git access token* or *Git access token secret* to continue adding a Git Repository. (link:https://issues.redhat.com/browse/OCPBUGS-7036[*OCPBUGS-7036*])
```
Comment (Srivaralakshmi) [inline release_notes/ocp-4-13-release-notes.adoc:1625]:
```suggestion
* Adding a Git Repository and configuring it with a GitLab and Bitbucket `pipeline-as-a-code` repository creates an invalid repository resource. As a result, the `spec.git_provider.url` Git provider URL is removed for GitLab and Bitbucket providers.

Workaround: Add the mandatory `spec.git_provider.user` field for Bitbucket. In addition, select either *Git access token* or *Git access token secret* to continue adding a Git Repository. (link:https://issues.redhat.com/browse/OCPBUGS-7036[*OCPBUGS-7036*])
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@shreyasiddhartha Nice revision! PTAL at my suggestions. Thanks!

**Review by Srivaralakshmi — APPROVED** [review]
Approved!

**Review by themr0c — APPROVED** [review]

**Pull request merged**